nautobot/extras/filters/__init__.py

```diff
@@ -297,6 +297,12 @@ class ConfigContextFilterSet(BaseFilterSet):
         to_field_name="slug",
         label="Tag (slug)",
     )
+    dynamic_group = NaturalKeyOrPKMultipleChoiceFilter(
+        field_name="dynamic_groups",
+        queryset=DynamicGroup.objects.all(),
+        to_field_name="slug",
+        label="Dynamic Group (slug or PK)",
+    )
```

Comment:
```suggestion
    dynamic_groups = NaturalKeyOrPKMultipleChoiceFilter(
        queryset=DynamicGroup.objects.all(),
        label="Dynamic Groups (slug or ID)",
    )
```

The other filters in this filterset are incorrectly named. The filter name should match the field name here.

Reply:
fixed